packages/hardhat-ts/package.json

```diff
@@ -1,63 +1,87 @@
-  "dependencies": {
-    "@eth-optimism/hardhat-ovm": "^0.2.4",
-    "@ethereumjs/vm": "^5.6.0",
-    "@ethersproject/providers": "^5.5.2",
-    "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@^0.3.0-beta.10",
-    "@nomiclabs/hardhat-waffle": "^2.0.1",
-    "@openzeppelin/contracts": "^4.4.2",
-    "@tenderly/hardhat-tenderly": "^1.0.13",
-    "@typechain/ethers-v5": "^9.0.0",
-    "@typechain/hardhat": "^4.0.0",
-    "chalk": "^4.1.2",
-    "ethereum-waffle": "^3.4.0",
-    "ethers": "^5.5.3",
-    "hardhat": "^2.8.2",
-    "hardhat-deploy": "^0.9.24",
-    "hardhat-deploy-ethers": "^0.3.0-beta.13",
-    "ipfs-http-client": "^54.0.2",
-    "qrcode-terminal": "^0.12.0",
-    "typechain": "^7.0.0"
-  },
-  "devDependencies": {
-    "@types/jest": "^27.4.0",
-    "@types/node": "^17.0.8",
-    "eslint": "^8.6.0",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-airbnb-typescript": "^16.1.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-jest": "^25.3.4",
-    "eslint-plugin-prettier": "^4.0.0",
-    "jest": "^27.4.7",
-    "node-watch": "^0.7.2",
-    "prettier": "^2.5.1",
-    "prettier-plugin-solidity": "^1.0.0-beta.13",
-    "rlp": "^2.2.7",
-    "solidity-coverage": "^0.7.17",
-    "ts-node": "^10.4.0",
-    "tsconfig-paths": "^3.12.0",
-    "typescript": "^4.5.4"
-  },
-  "installConfig": {
-    "hoistingLimits": "workspaces"
-  }
-}
+{
+  "name": "@scaffold-eth/hardhat",
+  "version": "0.1.0",
+  "license": "MIT",
+  "scripts": {
+    "chain": "hardhat node --network hardhat --no-deploy",
+    "fork": "hardhat node --fork https://mainnet.infura.io/v3/460f40a260564ac4a4f4b3fffb032dad",
+    "test": "hardhat test --network hardhat",
+    "compile": "hardhat compile",
+    "deploy": "hardhat deploy --export-all ../vite-app-ts/src/generated/hardhat_contracts.json",
+    "postdeploy": "hardhat run scripts/publish.ts",
+    "watch": "node scripts/watch.js",
+    "accounts": "hardhat accounts",
+    "balance": "hardhat balance",
+    "send": "hardhat send",
+    "generate": "hardhat generate",
+    "account": "hardhat account",
+    "etherscan-verify": "hardhat etherscan-verify --api-key PSW8C433Q667DVEX5BCRMGNAH9FSGFZ7Q8",
+    "lint": "eslint --config ./.eslintrc --ignore-path ./.eslintignore ./deploy/**/*.ts ./scripts/**/*.ts ./test/**/*.ts ./helpers/**/*.ts",
+    "solhint": "solhint ./contracts/**/*.sol",
+    "format": "yarn prettier -w ./deploy/**/*.ts ./scripts/**/*.ts ./test/**/*.ts ./helpers/**/*.ts contracts/**/*.sol ",
+    "format:check": "yarn prettier -c ./deploy/**/*.ts ./scripts/**/*.ts ./test/**/*.ts ./helpers/**/*.ts ./contracts/**/*.sol "
+  },
+  "dependencies": {
+    "@atixlabs/hardhat-time-n-mine": "^0.0.5",
+    "@defi-wonderland/smock": "^2.0.7",
+    "@eth-optimism/hardhat-ovm": "^0.2.4",
+    "@ethereum-waffle/ens": "^3.3.1",
+    "@ethereumjs/vm": "^5.6.0",
+    "@ethersproject/abi": "^5.5.0",
+    "@ethersproject/abstract-provider": "^5.5.1",
```

Comment:
This doesn't appear to be used anywhere, a usage of it was actually removed from `packages/vite-app-ts/src/functions/getNetworkInfo.ts`